src/middleware/rate_limit.rs

```diff
@@ -11,8 +11,6 @@ use std::{net::IpAddr, sync::Arc};
 use tokio::time::Duration;
 use tracing::error;
 
-pub const MAX_REQUESTS_PER_SEC: u32 = 10;
```

Comment:
Didn't rename to PER_MIN and removed instead since the desire to change rate limit is more than just changing this constant; we may change the window too